src/main/java/org/jabref/model/database/BibDatabase.java

```diff
@@ -584,6 +586,30 @@ private void relayEntryChangeEvent(FieldChangedEvent event) {
         eventBus.post(event);
     }
 
+    @Subscribe
+    private void listen(FieldAddedOrRemovedEvent event) {
+        System.out.println(event);
```

Comment:
Please use logging 😇 --> https://devdocs.jabref.org/getting-into-the-code/code-howtos.html#logging

Reply:
I'll just remove it. It's a leftover from debugging.